apps/astarte_housekeeping/test/support/database_test_helper.ex

```diff
@@ -38,7 +39,7 @@ defmodule Astarte.Housekeeping.DatabaseTestHelper do
   end
 
   def drop_astarte_keyspace do
-    query = "DROP KEYSPACE astarte"
+    query = "DROP KEYSPACE #{Config.astarte_instance_id!()}astarte"
```

Comment:
This should be computed using ` CQLUtils.realm_name_to_keyspace_name("astarte", Config.astarte_instance_id!())` rather than just interpolating the instance id